pkgs/by-name/ri/rido/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage {
+  pname = "rido";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "lj3954";
+    repo = "rido";
+    rev = "a1aa36a6108b34d2495256868b3b720170b8c306";
+    hash = "sha256-welK63Bbckr6/OrA2BQmuVPfYVhkOgBttDnC7BlzP4U=";
+  };
+
+  cargoHash = "sha256-yKr6HW0PaNx1ha6yCuimebEf8oG9K0ug8lX5Gq+Cx9o=";
+
+  meta = {
+    description = "Fetch valid URLs and checksums of Microsoft Operating Systems.";
+    homepage = "https://github.com/lj3954/rido";
+    license = lib.licenses.gpl3;
```

https://github.com/lj3954/rido/blob/a1aa36a6108b34d2495256868b3b720170b8c306/src/main.rs#L13
```suggestion
    license = lib.licenses.gpl3Plus;
```

---

Fixed by the last push.